crates/scroll/consensus/Cargo.toml

```diff
@@ -22,12 +22,10 @@ reth-consensus.workspace = true
 reth-consensus-common.workspace = true
 reth-ethereum-consensus.workspace = true
 reth-execution-types.workspace = true
-# imported in order to pull in the reth-codec feature for reth-ethereum-primitives
-reth-primitives = { workspace = true, features = ["reth-codec"] }
```

Comment:
this is legacy and can be removed, caused the `reth-codec` feature to be pulled unnecessarily.